enterprise/frontend/src/embedding-sdk/test/CommonSdkStoryWrapper.tsx

```diff
@@ -37,6 +37,7 @@ export const storybookSdkDefaultConfig: SDKConfig = {
       return response.json();
     } catch (e) {
       console.error("Failed to generate JWT", e);
+      return `Failed to generate JWT for storybook: ${e}`;
```

Comment:
This is the wrapper used in storybook, the returned string will be rendered in the error component in the `The ${source} must return an object with the shape {id:string, exp:number, iat:number, status:string}, got ${safeStringify(session)} instead` message.
This should make it a bit easier to debug, as previously we didn't return anything